includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -363,6 +379,12 @@ public function get_settings(): WP_REST_Response {
 				'is_platform_checkout_enabled'      => 'yes' === $this->wcpay_gateway->get_option( 'platform_checkout' ),
 				'platform_checkout_custom_message'  => $this->wcpay_gateway->get_option( 'platform_checkout_custom_message' ),
 				'platform_checkout_store_logo'      => $this->wcpay_gateway->get_option( 'platform_checkout_store_logo' ),
+				'deposit_schedule_interval'         => $this->wcpay_gateway->get_option( 'deposit_schedule_interval' ),
+				'deposit_monthly_anchor'            => $this->wcpay_gateway->get_option( 'deposit_schedule_monthly_anchor' ),
+				'deposit_weekly_anchor'             => $this->wcpay_gateway->get_option( 'deposit_schedule_weekhly_anchor' ),
```

Comment:
Typo: `deposit_schedule_weekhly_anchor`

```suggestion
				'deposit_weekly_anchor'             => $this->wcpay_gateway->get_option( 'deposit_schedule_weekly_anchor' ),
```